src/ReferenceTrimmerTask.cs

```diff
@@ -185,7 +185,7 @@ public override bool Execute()
                     .Where(path => !path.EndsWith("_._", StringComparison.Ordinal)) // Ignore special packages
                     .Select(path =>
                     {
-                        var packageFolderRelativePath = Path.Combine(nugetLibrary.Name, nugetLibrary.Version.ToNormalizedString(), path);
+                        var packageFolderRelativePath = Path.Combine(nugetLibrary.Name.ToLowerInvariant(), nugetLibrary.Version.ToNormalizedString(), path);
```

Comment:
Can you add a code comment explaining that this is what NuGet does? Ideally if you can find the place it does it that would be nice, but not necessary.

Reply:
Seems like the ToLowerInvariant also needs to be applied to version

Reply:
@jeffkl - what's the best way to mimic NuGet logic here to find the location of package artifacts in an OS-agnostic manner?

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/VersionFolderPathResolver.cs#L201

Reply:
The package path information is available in the lockfile, which is the crux of the latest iteration.